src/test/spec/json/uri-options/compression-options.json

```diff
@@ -7,12 +7,7 @@
       "warning": false,
       "hosts": null,
       "auth": null,
-      "options": {
-        "compressors": [
-          "zlib"
-        ],
-        "zlibCompressionLevel": 9
-      }
+      "options": {}
```

Comment:
Why were these removed -- was there an update to the spec tests that we pulled in here?

Reply:
The reason is that we only store compressors that are valid and enabled (via its corresponding feature flag).  When these tests are run none of the feature flags are set, so none of these compressors get stored.  As a result, when `ClientOptions` is deserialized, all these fields are empty.  We probably want to also run these tests with the feature flags set.

Reply:
I'll just disable these tests if not all of the feature flags are set.  I didn't realize that we don't manually edit these.